lib/modules/dosomething/dosomething_signup/dosomething_signup.signup_data_form.inc

```diff
@@ -612,3 +616,33 @@ function dosomething_signup_filter_form_submitted_copy($string, $timestamp) {
   $submitted = format_date($timestamp, 'custom', 'F jS');
   return str_replace(DOSOMETHING_SIGNUP_DATA_FORM_TIMESTAMP_TOKEN, $submitted, $string);
 }
+
+/**
+ * Sends a user to Gladiator (https://github.com/DoSomething/gladiator)
+ *
+ * @param object $user
+ *   The user that signup up for the competition.
+ * @param array $values
+ *   Values from the signup data form.
+ */
+function dosomething_signup_send_user_to_gladiator($user, $values) {
+  // Make sure the client is set up.
+  if (!_dosomething_signup_build_http_client()) { return; }
+
+  $client = _dosomething_signup_build_http_client();
```

Comment:
can probably do the if ($client) {return;} so you don't need to call `_dosomething_signup_build_http_client` twice